gammapy/modeling/tests/test_fit.py

```diff
@@ -221,7 +221,7 @@ def test_stat_profile():
     dataset.models.parameters["x"].scan_n_values = 3
     result = fit.stat_profile(datasets=[dataset], parameter="x")
 
-    assert_allclose(result["test.model.x_scan"], [0, 2, 4], atol=1e-7)
+    assert_allclose(result["test.x_scan"], [0, 2, 4], atol=1e-7)
```

Comment:
what is the naming logic here? does this solve the issue of whether `x` belongs to `model1` or `model2`?